test/ng/directive/booleanAttrsSpec.js

```diff
@@ -283,8 +284,9 @@ describe('ngHref', function() {
     expect(element.attr('href')).toEqual(undefined);
   }));
 
-  if (msie) {
-    // IE11/10/Edge fail when setting a href to a URL containing a % that isn't a valid escape sequence
+  // Support: IE 9-11 only, Edge 12-14+
+  if (msie || /\bEdge\/[\d\.]+\b/) {
```

Comment:
@jbedard Nice catch. This means, though, that we can just remove the `if` and run the test everywhere.

@Narretz A variable would make it easier to reuse and we should avoid UA-sniffing (it's more acceptable in tests but still); it's not robust, it can catch more browsers than intended and cause compatibility concerns. This is actually a good example as the `if` was unnecessary, I'll remove it.

IE sniffing is different as:

1. Browsers are trying to pretend they have nothing to do with IE; differently to e.g. Chrome which is imitated by lots of browsers.
2. We don't do it via the lying user agent string but the non-standard `document.documentMode` that other browsers will never implement.
3. There are lots of differences between IE & modern browsers and the gap will only get larger. Edge gets updated and doesn't lag so much so it won't apply to it.

IE sniffing is, therefore, one of the few safe browser sniffing available.